src/Workspaces/CSharp/Portable/LanguageServices/CSharpSemanticFactsService.cs

```diff
@@ -300,6 +300,9 @@ public bool IsPartial(ITypeSymbol typeSymbol, CancellationToken cancellationToke
                 semanticModel.GetDeclaredSymbol(memberDeclaration, cancellationToken));
         }
 
+        public IParameterSymbol FindParameterForArgument(SemanticModel semanticModel, SyntaxNode argumentNode, CancellationToken cancellationToken)
+            => ((ArgumentSyntax)argumentNode).DetermineParameter(semanticModel, allowParams: false, cancellationToken);
```

Comment:
added to help out the regex/json features (which want to see if they're used in situatiosn like JToken.Parse and new Regex).